Descent3/fireball.cpp

```diff
@@ -1091,7 +1092,7 @@ void CreateSplintersFromBody(object *obj, float explosion_mag, float lifetime) {
     dest = center + obj->pos;
 
     // Now create splinter with that faces center position
-    int s_objnum = ObjCreate(OBJ_SPLINTER, pm - Poly_models, obj->roomnum, &dest, NULL);
+    int s_objnum = ObjCreate(OBJ_SPLINTER, pm - std::data(Poly_models), obj->roomnum, &dest, NULL);
```

Comment:
possible to use similar pattern as `OBJNUM`?

Reply:
This is merely adapting the code to function. Anyway, the OBJNUM pattern is too hacky anyway. Moving to an iterator based approach would be better. I'm just trying to get the process started.